src/main/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionCategoryServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package pl.byczazagroda.trackexpensesappbackend.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import pl.byczazagroda.trackexpensesappbackend.dto.FinancialTransactionCategoryDTO;
+import pl.byczazagroda.trackexpensesappbackend.dto.FinancialTransactionCategoryUpdateDTO;
+import pl.byczazagroda.trackexpensesappbackend.exception.AppRuntimeException;
+import pl.byczazagroda.trackexpensesappbackend.exception.ErrorCode;
+import pl.byczazagroda.trackexpensesappbackend.mapper.FinancialTransactionCategoryModelMapper;
+import pl.byczazagroda.trackexpensesappbackend.model.FinancialTransactionCategory;
+import pl.byczazagroda.trackexpensesappbackend.repository.FinancialTransactionCategoryRepository;
+
+import javax.transaction.Transactional;
+
+@Service
+@RequiredArgsConstructor
+public class FinancialTransactionCategoryServiceImpl implements FinancialTransactionCategoryService {
+
+    private final FinancialTransactionCategoryRepository financialTransactionCategoryRepository;
+
+    private final FinancialTransactionCategoryModelMapper financialTransactionCategoryModelMapper;
+
+    @Override
+    @Transactional
+    public FinancialTransactionCategoryDTO updateFinancialTransactionCategory(
```

Comment:
We need to speak about validation annotations:
`@Min(1)`, `@NotNull`, `@Valid` 
on the service application layer.

For now, you got it on the service interface and you got it on Controller.